src/openvic-simulation/politics/Government.hpp

```diff
@@ -0,0 +1,42 @@
+#pragma once
+
+#include "openvic-simulation/types/IdentifierRegistry.hpp"
+#include "openvic-simulation/politics/Ideology.hpp"
+
+namespace OpenVic {
+    struct GovernmentTypeManager;
+
+    struct GovernmentType : HasIdentifier {
+        friend struct GovernmentTypeManager;
+
+    private:
+        std::vector<std::string> ideologies;
```

Comment:
```suggestion
        std::vector<Ideology const*> ideologies;
```
Where possible we should provide pointers to things to avoid future identifier lookups.

Reply:
Of course, this requires loading the ideologies first. But the ideologies themselves can contain arbitrary trigger conditions for upperhouse support. And a trigger condition could in turn refer to one of the government types by name ...

Reply:
We'll have to keep the node trees for conditions and effects in memory until after standard loading so they can be parsed properly, just have to move the unique node pointer to a temporary variable so it doesn't get destroyed.